CHANGELOG.md

```diff
@@ -220,18 +222,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Intersection Joiner
 - Geoparquet Loader
 
-[unreleased]: https://github.com/srai-lab/srai/compare/0.6.1...HEAD
-[0.6.1]: https://github.com/srai-lab/srai/compare/0.6.0...0.6.1
-[0.6.0]: https://github.com/srai-lab/srai/compare/0.5.2...0.6.0
-[0.5.2]: https://github.com/srai-lab/srai/compare/0.5.1...0.5.2
-[0.5.1]: https://github.com/srai-lab/srai/compare/0.5.0...0.5.1
-[0.5.0]: https://github.com/srai-lab/srai/compare/0.4.1...0.5.0
-[0.4.1]: https://github.com/srai-lab/srai/compare/0.4.0...0.4.1
-[0.4.0]: https://github.com/srai-lab/srai/compare/0.3.3...0.4.0
-[0.3.3]: https://github.com/srai-lab/srai/compare/0.3.2...0.3.3
-[0.3.2]: https://github.com/srai-lab/srai/compare/0.3.1...0.3.2
-[0.3.1]: https://github.com/srai-lab/srai/compare/0.3.0...0.3.1
-[0.3.0]: https://github.com/srai-lab/srai/compare/0.2.0...0.3.0
-[0.2.0]: https://github.com/srai-lab/srai/compare/0.1.1...0.2.0
-[0.1.1]: https://github.com/srai-lab/srai/compare/0.0.1...0.1.1
-[0.0.1]: https://github.com/srai-lab/srai/compare/687500b...0.0.1
+[Unreleased]: https://github.com/kraina-ai/srai-sandbox/compare/0.6.1...HEAD
+
+[0.6.1]: https://github.com/kraina-ai/srai-sandbox/compare/0.6.0...0.6.1
+
+[0.6.0]: https://github.com/kraina-ai/srai-sandbox/compare/0.5.2...0.6.0
+
+[0.5.2]: https://github.com/kraina-ai/srai-sandbox/compare/0.5.1...0.5.2
+
+[0.5.1]: https://github.com/kraina-ai/srai-sandbox/compare/0.5.0...0.5.1
+
+[0.5.0]: https://github.com/kraina-ai/srai-sandbox/compare/0.4.1...0.5.0
+
+[0.4.1]: https://github.com/kraina-ai/srai-sandbox/compare/0.4.0...0.4.1
+
+[0.4.0]: https://github.com/kraina-ai/srai-sandbox/compare/0.3.3...0.4.0
+
+[0.3.3]: https://github.com/kraina-ai/srai-sandbox/compare/0.3.2...0.3.3
+
+[0.3.2]: https://github.com/kraina-ai/srai-sandbox/compare/0.3.1...0.3.2
+
+[0.3.1]: https://github.com/kraina-ai/srai-sandbox/compare/0.3.0...0.3.1
+
+[0.3.0]: https://github.com/kraina-ai/srai-sandbox/compare/0.2.0...0.3.0
+
+[0.2.0]: https://github.com/kraina-ai/srai-sandbox/compare/0.1.1...0.2.0
+
+[0.1.1]: https://github.com/kraina-ai/srai-sandbox/compare/0.0.1...0.1.1
+
+[0.0.1]: https://github.com/kraina-ai/srai-sandbox/releases/tag/0.0.1
```

Comment:
Urls changed to the sandbox repo - to fix

Reply:
Good point! Thanks

Reply:
done